services/api/src/models/definitions/audit-log.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I usually try to avoid model names that don't pluralize well. The name of this model is plural.

I recommend using a singular noun that can be pluralized:

- `audit-trail-entry` / `audit-trail-entries`
- `audit-log-entry` / `audit-log-entries`

Reply:
isnt `logs` the plural form? log would be singular.

if you dont like log what about record? 

i prefer the plural from is just adding an `s` to the end + short